neuraxle/pipeline.py

```diff
@@ -70,22 +71,15 @@ def transform(self, data_inputs: Any):
         :param data_inputs: the data input to transform
         :return: transformed data inputs
         """
-        data_container = DataContainer(data_inputs=data_inputs, current_ids=None)
-
-        data_container = self.hash_data_container(data_container)
-        context = ExecutionContext(root=self.cache_folder, execution_mode=ExecutionMode.TRANSFORM)
-        context = context.push(self)
-        data_container = self._transform_data_container(data_container, context)
-
+        data_container = self.transform_data_container(DataContainer(data_inputs=data_inputs, current_ids=None))
         return data_container.data_inputs
 
     def transform_data_container(self, data_container: DataContainer):
-        data_container = self.hash_data_container(data_container)
-        context = ExecutionContext(root=self.cache_folder, execution_mode=ExecutionMode.TRANSFORM)
-        context = context.push(self)
-        data_container = self._transform_data_container(data_container, context)
-
-        return data_container.data_inputs
+        data_container = self.handle_transform(
+            data_container,
+            ExecutionContext(root=self.cache_folder, execution_mode=ExecutionMode.TRANSFORM)
+        )
```

Comment:
is the sklearn wrapper always using a weird new context ? thats probably ok... I didn't realize this before.

Reply:
it has always been like that. I think that is fine. resolving